controllers/array_action/array_mediator_svc.py

```diff
@@ -1505,6 +1520,22 @@ def _is_vdisk_has_fcmaps(self, vdisk_uid):
         cli_volume = self._get_cli_volume_by_wwn(vdisk_uid, not_exist_err=False)
         return cli_volume and cli_volume.FC_id
 
+    def _mkhost(self, host_name, connectivity_type, ports):
+        cli_kwargs = build_create_host_kwargs(host_name, connectivity_type, ports)
+        try:
+            self.client.svctask.mkhost(**cli_kwargs)
+        except (svc_errors.CommandExecutionError, CLIFailureError) as ex:
```

Comment:
what if it's just a warning message?
could you ensure there's a ticket to make the warning check a decorator, and use it for all `svctask`s?

Reply:
done - #561

Reply:
CSI-4604